models/ideas.js

```diff
@@ -19,10 +19,18 @@ function createIdea(header, description, askedSum, userId) {
 }
 
 function updateIdea(ideaId, key, value) {
+    if (!value) {
+        throw new Error(`Error. You didn't provide any value to update`);
```

Comment:
You're missing `.` at the end of the error message.

Keep the style of your code consistent.

Reply:
Updated. Commited. Tested.